app/components/common/icons/arrows/PrevAndNext.js

```diff
@@ -0,0 +1,54 @@
+export default function PrevAndNext() {
```

Comment:
https://softwareengineering.stackexchange.com/questions/402979/what-does-it-mean-for-a-method-or-a-function-to-do-one-thing/402981#402981

break up PrevAndNext into two components